src/components/shared/header-component/Header/DesktopHeader.jsx

```diff
@@ -25,7 +26,8 @@ const DesktopHeader = ({ intl }) => {
   const allowPublicAccountCreation = getConfig().ALLOW_PUBLIC_ACCOUNT_CREATION ?? true;
   const history = useHistory();
   const { authenticatedUser } = useContext(AppContext);
-  const { headerLogo, loading } = useGetConfig();
+  const { headerLogo, loading, useTPAOnly, TPAQueryparam } = useGetConfig();
```

Comment:
in terms of best-practice, the 'use' prefix is only usable for hooks so please change `useTPAOnly` to something like `isTPAOnly`